src/pages/Editor/sections/OutputComponent/Output.js

```diff
@@ -57,6 +57,7 @@ const OutputComponent = () => {
               color: `${editContext.bodyValues.bodyColor}`,
               width: `${editContext.bodyValues.bodyWidth}pc`,
               letterSpacing: `${editContext.bodyValues.bodyLetterSpace}px`,
+              overflowY : `scroll`,
```

Comment:
```suggestion
              overflowY : 'scroll',
```
Why're you making use of template literals, if you don't want to embed expression?

Reply:
Will keep it in my mind in the future, Thank You.